qiskit_experiments/library/stark/__init__.py

```diff
@@ -0,0 +1,30 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+===============================================================================================
+Stark Experiments (:mod:`qiskit_experiments.library.stark`)
```

Comment:
The purpose of this module is to provide experiment to deal with TLS. I don't think we will have many experiments in this module. So far I have implemented RamseyXY, RamseyXY with amplitude scan at fixed delay (StarkRamseyFast), and P1 spectroscopy. Probably we can put these experiments in the standard characterization module.

Reply:
I removed the dedicated module with [56349b8](https://github.com/Qiskit/qiskit-experiments/pull/1008/commits/56349b8ed6d7e9520dca17fbd0d0da7a4052893c).

Reply:
Okay, that will probably be fine. I was not opposed to a new directory if there were enough experiments for it. I was wondering if we could scope it to a topic of study (like TLS, or maybe frequency shifting?) rather than a technique like Stark.